tests/data/bundle-large_whitelist.json

```diff
@@ -13470,6 +13470,20 @@
       ],
       "name": "Global Whitelist",
       "version": "1_0"
+    },
+    {
```

Comment:
Great that this data is here, but is this used in an actual test?

Reply:
as discussed, used for implicit schema validation and execution